INSTALL.rst

```diff
@@ -320,3 +320,24 @@ __ https://packaging.python.org/en/latest/guides/installing-using-pip-and-virtua
 .. _PATH: `Configuring path`_
 .. _PyPI: https://pypi.org/project/robotframework
 .. _GitHub: https://github.com/robotframework/robotframework
+
+
+
+
+Zipapp 
+--------------------
+Zipapps (https://docs.python.org/3/library/zipapp.html) are technique to distribute all the python code of  asolution in a single file, which can be executed using a python interpreter. The same zipapp file can be run on multiple plattforms. An example of using pdm 
+(https://pdm.fming.dev/latest/) with the packer extension (https://github.com/frostming/pdm-packer) to create a zipapp would be.
+
+.. sourcecode:: bash
+
+    $ pdm init 
+    $ pdm add robotframework
+    $ #If the target is python 3.6: pdm add importlib_resources         
+    $ pdm pack -m robot:run_cli
```

Comment:
Is `pdm` needed to be able to use zipapps for anything useful? If not, I'd prefer an example using just the standard `zipapp` here. There could be a separate example using `pdm`, but perhaps an external project with more documentation that could be referenced here would be even better.

Reply:
No pdm is not needed, but it is the tool I use on a daily basis for a reason....

venv should also work, also shiv or pex...  poetry I would need to ask around... but completely without tool support zipapps are not practical. I am happy if we just state the option and link to marketsquare....